src/Features/SupportConsoleCommands/Tests/MakeCommandUnitTest.php

```diff
@@ -39,7 +39,7 @@ public function component_test_is_created_by_make_command_with_test_option()
     /** @test */
     public function component_pest_test_is_created_by_make_command_with_pest_option()
     {
-        Artisan::call('make:livewire', ['name' => 'foo', '--pest' => true]);
+        Artisan::call('make:livewire', ['name' => 'foo', '--test' => true, '--pest' => true]);
```

Comment:
Before if both flags were present it created the test with the PHPUnit scaffold, and now if the `--pest` flag is present it will create the test with the current scaffold even if the `--test` flag is not there